.buildkite/scripts/e2e/bootstrap.py

```diff
@@ -0,0 +1,164 @@
+            for line in infile:
+                if "stack.logstash_enabled: true" in line:
+                    # Logstash is disabled by default, remove the comment
+                    line = line.lstrip('#').lstrip()
+                outfile.write(line)
+
+    def __setup_elastic_package_profile(self) -> None:
+        Util.run_or_raise_error(["elastic-package", "profiles", "create", "e2e"],
+                                "Error occurred while creating a profile. Check logs for details.")
+
+        print("`elastic-package` e2e profile created.")
+
+        # elastic-package creates a profile under home directory
+        config_example_file = os.path.join(self.__get_profile_path(), "config.yml.example")
+        config_file = os.path.join(self.__get_profile_path(), "config.yml")
+        self.__create_config_file(config_example_file, config_file)
+        Util.run_or_raise_error(["elastic-package", "profiles", "use", "e2e"],
+                                "Error occurred while creating a profile. Check logs for details.")
+
+    def __install_plugin(self) -> None:
+        with open("VERSION", "r") as f:
+            version = f.read()
+
+        plugin_name = f"logstash-filter-elastic_integration-{version.strip()}-java.gem"
+        Util.run_or_raise_error(["docker", "cp", plugin_name, f"{self.LOGSTASH_CONTAINER_NAME}:/usr/share/logstash"],
+                                "Error occurred while copying plugin to container, see logs for details.")
+
+        print("Installing logstash-filter-elastic_integration plugin...")
+        Util.run_or_raise_error(
+            ["docker", "exec", self.LOGSTASH_CONTAINER_NAME, "bin/logstash-plugin", "install", plugin_name],
+            "Error occurred installing plugin in Logstash container")
+        print("Plugin installed successfully.")
+
+    def __reload_container(self) -> None:
+        print("Restarting Logstash container after installing the plugin.")
+        Util.run_or_raise_error(["docker", "restart", f"{self.LOGSTASH_CONTAINER_NAME}"],
+                                "Error occurred while reloading Logstash container, see logs for details.")
+        time.sleep(20)  # give a time Logstash pipeline to fully start
+
+    def __update_pipeline_config(self) -> None:
+        local_config_file_path = ".buildkite/scripts/e2e/config/"
+        config_file = "serverless_pipeline.conf" if self.project_type == "serverless" else "pipeline.conf"
+        local_config_file = local_config_file_path + config_file
+        container_config_file_path = "/usr/share/logstash/pipeline/logstash.conf"
+        # python docker client (internally uses subprocess) requires special TAR header with tar operations
+        Util.run_or_raise_error(["docker", "cp", f"{local_config_file}",
+                                 f"{self.LOGSTASH_CONTAINER_NAME}:{container_config_file_path}"],
+                                "Error occurred while replacing pipeline config, see logs for details")
```

Comment:
Where can the logs be found?

I encountered:

```
  File "/home/dl/source/elastic/logstash-filter-elastic_integration/.buildkite/scripts/e2e/util.py", line 35, in run_or_raise_error
    raise Exception(f"{full_error_message}")
Exception: Error occurred while replacing pipeline config, see logs for details
```

Reply:
This will be covered by https://github.com/elastic/elastic-package/pull/1668
Long story short, we need changeable pipeline configuration for long term and I am pushing it with this change.
I will update once it gets landed.

Reply:
New version (0.98.0) of `elastic-package` is released and now this is unblocked.